Sources/OpenAPIRuntime/Interface/UniversalClient.swift

```diff
@@ -90,76 +90,116 @@ import Foundation
         serializer: @Sendable (OperationInput) throws -> (HTTPRequest, HTTPBody?),
         deserializer: @Sendable (HTTPResponse, HTTPBody?) async throws -> OperationOutput
     ) async throws -> OperationOutput where OperationInput: Sendable, OperationOutput: Sendable {
-        @Sendable
-        func wrappingErrors<R>(
+        @Sendable func wrappingErrors<R>(
```

Comment:
Is this a result of changing formatting options or just a manual fix up?

Reply:
Manual, I believe, but soon when respectExistingLineBreaks is turned off in the runtime library (as it was recently in the generator), it won't matter, as it'll enforce one style regardless of how it was originally written. And I think swift-format will put it on a single line.